lms/djangoapps/mobile_api/testutils.py

```diff
@@ -92,9 +92,9 @@ def reverse_url(self, reverse_args=None, **kwargs):
             reverse_args.update({'username': kwargs.get('username', self.user.username)})
         return reverse(self.REVERSE_INFO['name'], kwargs=reverse_args)
 
-    def url_method(self, url, **kwargs):  # pylint: disable=unused-argument
+    def url_method(self, url, qargs={}, **kwargs):  # pylint: disable=unused-argument
         """Base implementation that returns response from the GET method of the URL."""
-        return self.client.get(url)
+        return self.client.get(url, qargs)
```

Comment:
Also, it looks like the django client uses the keyword `"data"` for this argument instead of `"qargs"`.  For consistency and to lower the cognitive load on developers reading the code, we should do the same.